src/components/Header/Header.tsx

```diff
@@ -0,0 +1,143 @@
+                        />
+                    </svg>
+                </a>
+
+                <div className={styles.headerThemeModeBtns}>
+                    <button className={styles.headerThemeModeBtn}>
+                        <svg
+                            className={styles.headerThemeModeBtnIcon}
+                            xmlns="http://www.w3.org/2000/svg"
+                            width="20"
+                            height="20"
+                            viewBox="0 0 20 20"
+                        >
+                            <g clipPath="url(#clip0_3001_1631)">
+                                <path
+                                    d="M10.0001 3.33329V1.66663M10.0001 16.6666V18.3333M5.34526 5.34514L4.16675 4.16663M14.7734 14.7733L15.952 15.9518M3.33341 9.99996H1.66675M16.6667 9.99996H18.3334M14.7738 5.34514L15.9524 4.16663M5.34567 14.7733L4.16716 15.9518M10.0001 14.1666C7.69889 14.1666 5.83341 12.3011 5.83341 9.99996C5.83341 7.69877 7.69889 5.83329 10.0001 5.83329C12.3013 5.83329 14.1667 7.69877 14.1667 9.99996C14.1667 12.3011 12.3013 14.1666 10.0001 14.1666Z"
+                                    strokeWidth="1.5"
+                                    strokeLinecap="round"
+                                    strokeLinejoin="round"
+                                />
+                            </g>
+                        </svg>
+                    </button>
+
+                    <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 2 20">
+                        <path d="M1 0V20" stroke="#656565" />
+                    </svg>
+
+                    <button className={[styles.headerThemeModeBtn, styles.headerThemeModeBtnActive].join(' ')}>
+                        <svg
+                            className={styles.headerThemeModeBtnIcon}
+                            xmlns="http://www.w3.org/2000/svg"
+                            width="20"
+                            height="20"
+                            viewBox="0 0 20 20"
+                        >
+                            <path
+                                d="M7.5 4.99996C7.5 9.14209 10.8579 12.5 15 12.5C15.7577 12.5 16.4892 12.3879 17.1787 12.1789C16.2453 15.2585 13.3844 17.4998 10 17.4998C5.85786 17.4998 2.5 14.1422 2.5 10.0001C2.5 6.61565 4.74173 3.75482 7.82134 2.82141C7.61236 3.51089 7.5 4.24224 7.5 4.99996Z"
+                                strokeWidth="1.5"
+                                strokeLinecap="round"
+                                strokeLinejoin="round"
+                            />
+                        </svg>
+                    </button>
+                </div>
+
+                <div className={styles.headerPageLinks}>
+                    <a className={styles.headerPageLink} href="#about">
```

Comment:
Add, please active state for this link.
Would be good to change cursor for the active item here and for the theme switcher to `default` or `none`. It will inform a user, that click on this element will do nothing.

Reply:
Fixed